src/ParseSchema.js

```diff
@@ -425,34 +363,30 @@ class ParseSchema {
 }
 
 const DefaultController = {
-  send(className: string, method: string, params: any, options: RequestOptions): Promise {
+  send(className: string, method: string, params: any): Promise {
     const RESTController = CoreManager.getRESTController();
-    const requestOptions = { useMasterKey: true };
-    if (options.hasOwnProperty('sessionToken')) {
-      requestOptions.sessionToken = options.sessionToken;
-    }
     return RESTController.request(
       method,
       `schemas/${className}`,
       params,
-      requestOptions
+      { useMasterKey: true }
     );
   },
 
-  get(className: string, options: RequestOptions): Promise {
-    return this.send(className, 'GET', {}, options);
+  get(className: string): Promise {
+    return this.send(className, 'GET', {});
```

Comment:
what happens if you just do `this.send(className, 'GET')`
or `this.send(className, 'DELETE')`

would be nice if send set the default param of {} instead of using here?